### ISSUE TITLE
Flag provided but not defined

### ISSUE THREAD

**Issue by nodeit** [express42/vagrant-foodshow#5]
~~~
config.foodshow.enabled = true
config.foodshow.subdomain = 'MYSUBDOMAINHERE'
config.foodshow.authtoken = 'MYAUTHTOKENHERE'
config.vm.network :forwarded_port, guest: 80, host: 8082, ngrok_proto: "http+https"
~~~

Above is my Vagrantfile setup for vagrant-foodshow and I am receiving the following error:

    Incorrect Usage: flag provided but not defined: -log

**Comment by sample** [comment]
Which version of ngrok you are using?

**Comment by nodeit** [comment]
2.0.17

**Comment by pittaya** [comment]
using ngrok 2.0.17 on OS X 10.10.3 and got the same error.

**Comment by nodeit** [comment]
I'm also on OS X 10.10.

**Comment by sample** [comment]
Current version of vagrant-food show doesn't work with ngrok 2.x. I'll add ngrok2 support in future.

**Comment by nodeit** [comment]
No rush, just curious if you had a timeframe in mind for ngrok2 support?

**Comment by sample** [comment]
ngrok v2 support will be available next week.

**Comment by nodeit** [comment]
You rock, thanks!

**Comment by sample** [comment]
Not me :) https://github.com/express42/vagrant-foodshow/pull/6

**Comment by nodeit** [comment]
Well then you just get partial credit ;)

**Comment by sample** [comment]
Released v1.1.0 with ngrok v2 support.
